modules/AWS/eks/autoscaling.tf

```
@@ -0,0 +1,102 @@
+module "autoscaler_iam_role" {
+  source  = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
+  version = "4.13.2"
+
+  create_role  = true
+  role_name    = "${var.cluster_name}-cluster-autoscaler"
```

cluster name has `-cluster` as postfix so we don't have to repeat it here. 
```suggestion
  role_name    = "${var.cluster_name}-autoscaler"
```